docs/buildah-push.md

```diff
@@ -95,6 +99,10 @@ This example extracts the imageID image to a container registry named registry.e
 
  `# buildah push imageID docker://registry.example.com/repository:tag`
 
+This example extracts the imageID image to a container registry named registry.example.com and saves the digest ID in the specified didigestfile.
```

Comment:
“the digest”, or maybe “the digest value”, not “the digest ID”, please.

Also see above WRT saying that it is a digest at the destination.

Reply:
changed to "saves the digest"